code/game/objects/machinery/bots/bots.dm

```diff
@@ -110,27 +112,27 @@
 			else
 				bot_startup()
 
-/obj/machinery/bot/proc/bot_shutdown()
+/obj/machinery/bot/proc/bot_shutdown(mob/living/user)
 	balloon_alert_to_viewers("Powers off")
 	if(deactivation_animation)
 		flick("[deactivation_animation]", src)
 	if(length(shutdownsentences))
 		say(pick(shutdownsentences))
 	is_active = FALSE
-	update_icon_state()
+	update_icon()
 	stop_processing()
 
-/obj/machinery/bot/proc/bot_startup()
+/obj/machinery/bot/proc/bot_startup(mob/living/user)
 	balloon_alert_to_viewers("Powers on")
 	if(activation_animation)
 		flick("[activation_animation]", src)
 	if(length(awakeningsentences))
 		say(pick(awakeningsentences))
 	is_active = TRUE
-	update_icon_state()
+	update_icon()
 	start_processing()
 
-/obj/machinery/bot/update_icon_state()
+/obj/machinery/bot/update_icon()
```

Comment:
update_icon_state was right, it's just the outside calls which should be update_icon instead of calling update state directly.